src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss

```diff
@@ -0,0 +1,68 @@
+.flex-microform {
```

Comment:
pls reduce the number of styles here, are they all really necessary?

Reply:
Kindly provide more details on this.

Reply:
It seemed to me, some style definition are not necessary here, e.g. is it really necessary to define the font-stretch, e.g.? So I wanted you to check this. If you think, we need all these styles leave it as it is now.